generators/lib/distros/openatrium.js

```diff
@@ -20,9 +21,17 @@ function init() {
   module.description = 'This project is built on [' + module.option.name + '](http://openatrium.com) for more information visit the [Atrium Project Homepage](https://drupal.org/project/openatrium).';
 
   module.releaseVersion = function(majorVersion, done, cb) {
-    require('../../lib/drupalProjectVersion').latestReleaseStable(module.id, majorVersion, done, cb);
+    require('../drupalProjectVersion').latestReleaseStable(module.id, majorVersion, done, cb);
   };
 
+  module.loadComposer = function(yo, options) {
+    var file = yo.templatePath('drupal/' + module.id + '/' + options.drupalDistroVersion + '/composer.json');
```

Comment:
You are right, Atrium and Drupal have the same code for this. I'd like to see what kind of inheritance model we might use for the distribution plugins. That's another good follow-up.